reapy/core/project/marker.py

```diff
@@ -7,18 +7,18 @@ class Marker(ReapyObject):
 
     _class_name = "Marker"
 
-    def __init__(
-        self, parent_project=None, index=None, parent_project_id=None
-    ):
```

Comment:
As far as I know, this is a debate that has not officially been closed by any PEP (see [this question](https://stackoverflow.com/questions/24988162/define-functions-with-too-many-arguments-to-abide-by-pep8-standard)). I rather agree with the accepted answer: I prefer consistent indentation levels rather than arguments packed together at the end of the line. I'm not sure it's a good idea to start changing all of these in the whole project now... Unless you have a good reason I'm unaware of! This comment applies to several changes you have made in the code.